prawns/tips/src/handlers/tips.tip.extrinsic.ts

```diff
@@ -1,29 +1,35 @@
 import { u8aToHex } from '@polkadot/util';
-import { ExtrinsicHandlerContext } from '@subsquid/substrate-processor';
+import { CallHandlerContext } from '@subsquid/substrate-processor';
+import { Store } from '@subsquid/typeorm-store';
+import assert from 'assert';
 import { SubstrateNetwork, SubstrateTip, SubstrateTipper } from '../model';
-import { decodeAddress } from '../utils';
+import { decodeAddress, getCallOriginAccount } from '../utils';
 import { getTipsTipCall } from './typeGetters/getTipsTipCall';
 
 export default (network: SubstrateNetwork) =>
-  async (ctx: ExtrinsicHandlerContext) => {
+  async (ctx: CallHandlerContext<Store>) => {
     const tipCall = getTipsTipCall(ctx, network);
-    const blockNumber = BigInt(ctx.block.height);
-    const account = ctx.extrinsic.signer;
-    const rootAccount = decodeAddress(account);
-    const date = new Date(ctx.block.timestamp);
 
     const tipModel = await ctx.store.get(SubstrateTip, u8aToHex(tipCall.hash));
     if (!tipModel) {
       // NOTE: We don't break here because there can be old tippers or tips that are created through the treasury.NewTip.
       // For now we don't need to index those events.
-      console.log(`tips.tip.extrinsic::Tip not found: ${u8aToHex(tipCall.hash)}`);
+      console.log(
```

Comment:
I saw we have moved to use ctx.log for one other log. I think we should be consistent of either console or use their logger.